vdirsyncer/cli/utils.py

```diff
@@ -4,6 +4,7 @@
 import json
 import os
 import sys
+from typing import Any
```

Comment:
I think we also need `from __future__ import annotations` in this file for compat with older Pythons.

This likely applies to other files touched in the PR too.

Reply:
CI failures are because of this: https://builds.sr.ht/~whynothugo/job/1045797#task-test